users/views.py

```diff
@@ -216,3 +217,31 @@ def post(self, request):
             return JsonResponse({'MESSAGE': 'KEY_ERROR'}, status=400)
         except TypeError:
             return JsonResponse({'MESSAGE': 'TYPE_ERROR'}, status=400)
+
+class UserProductLikeView(View):
+    @utils.login_required
+    def post(self, request):
+        try:
+            data    = json.loads(request.body)
+            user    = getattr(request,'user',None)
+            product = Product.objects.get(id=data['product'])
+            like   = UserProductLike.objects.filter(user=user, product=product).first()
+
+            if like:
+                like.is_like = not like.is_like
+                like.save()
+                
+                return JsonResponse({'LIKE': like.is_like}, status=200)
+            UserProductLike.objects.create(user=user, product=product, is_like=True)
+
+            return JsonResponse({'LIKE': True}, status=200)
```

Comment:
```suggestion

            if like:
                like.is_like = not like.is_like
                like.save()                
                return JsonResponse({'LIKE': like.is_like}, status=200)
                
            UserProductLike.objects.create(user=user, product=product, is_like=True)
            return JsonResponse({'LIKE': True}, status=200)
```

good!
근데 로직별로 라인 분리 수정 안되신 것 같아요 :)